zhaquirks/develco/__init__.py

```diff
@@ -9,3 +14,66 @@ class DevelcoPowerConfiguration(PowerConfigurationCluster):
 
     MIN_VOLTS = 2.6  # old 2.1
     MAX_VOLTS = 3.0  # old 3.2
+
+
+class DevelcoIasZone(CustomCluster, IasZone):
```

Comment:
I think this class only needs to define the `attributes` dict and the `server_commands` dict. Everything else should be inherited. @Adminiuga that sound right? I think this is another device that doesn't follow the spec for the IAS notifications.

Reply:
I'm pretty sure I tried to remove the ZoneType and ZoneState (because that is actually the same as spec) and it was throwing an error. Happy to take some pointers of what to do there, because I probably most definitely missed something obvious!

The ZoneStatus is actually different to spec according to the datasheet from Develco so that needs included I believe.

Reply:
No need to redefine everything. Just inherit the IasZone cluster and change schema for the commands only or define those as manufacturer_server_commands.

Can you also post a debug log with incoming message containing that command? Usually there's extended status too